test/manual/partition_data_test.cc

```diff
@@ -61,7 +61,11 @@ BOOST_AUTO_TEST_CASE(test_atomic_cell) {
     };
 
     for (auto tc : cases) {
-        auto [live, fixed_size, value, expiring, counter_update] = tc;
+        auto& live = tc.live;
+        auto& fixed_size = tc.fixed_size;
+        auto& value = tc.value;
+        auto& expiring = tc.expiring;
+        auto& counter_update = tc.counter_update;
```

Comment:
Shouldn't it be auto instead of auto&?

Reply:
No, why?

Reply:
Again, just pointing out the change in behavior. Variables were lvalues and now are lvalue references. It's probably fine to change it like this.

Reply:
Yes, these sort of changes bothered me too. I have no practical way to review this change without trying to read the rest of the file and what is actually done with these references - which I assume you (Avi) did, but I didn't.

Reply:
The variables were actually structured bindings, which isn't the same as a free standing variable (this is why it couldn't be captured in a lambda). A reference is the closest thing to a structured binding. Note also the container of the data is in a regular variable, so should have the same lifetime.